ci/run-tests.ps1

```diff
@@ -17,18 +18,18 @@ function Force-ResolvePath {
     return $ExecutionContext.SessionState.Path.GetUnresolvedProviderPathFromPSPath($path)
 }
 
-if ($override_spatial_networking) {
+if ($run_with_spatial) {
     # Generate schema and snapshots
-    Echo "Generating snapshot and schema for testing project"
-    $commandlet_process = Start-Process "$unreal_editor_path" -Wait -PassThru -NoNewWindow -ArgumentList @(`
+    Write-Output "Generating snapshot and schema for testing project"
+    Start-Process "$unreal_editor_path" -Wait -PassThru -NoNewWindow -ArgumentList @(`
         "$uproject_path", `
```

Comment:
How come the other lines have had the backtick removed but this one hasn't.

Reply:
I think the comments function similarly to the backtick, to escape the line ending.